oggm/utils.py

```diff
@@ -218,12 +245,17 @@ def _download_alternate_topo_file(fname):
     """
 
     # temporary check
-    fzipname = fname + '.zip'
+    fzipname = fname.split('.')[0] + '.zip'
     if fzipname not in [
-        'gimpdem_90m.tif.zip',
-        'iceland.tif.zip',
-        'northcanada.tif.zip',
-        'svalbard.tif.zip',
+        'gimpdem_90m.zip',
+        'iceland.zip',
+        'northcanada.zip',
+        'svalbard.zip',
+        'AN1.zip',
+        'AN2.zip',
+        'AN3.zip',
+        'AN4.zip',
+        'AN5.zip'
     ]:
```

Comment:
Ah okay!
I thought it was easier to have all SRTM DEMs from one source (as they should be the same anyway!?)
I will change things the way you have suggested now over the next days.
I think a good name for the dataset was `_download_dem3_filled` if you are okay with it.

AN1 to AN5 stand for the Antarctica tiles 1-5.

Last but not least: yes, if you talk about the `fzipname` test, we should definitely check whether we can implement special conditions for special lats/lons in the `_download_dem3_filled` download function. That's what I believe would be the easiest to get the files nevertheless.